src/core/geometry/fuzzyfactory/FuzzyFactory2d.js

```diff
@@ -1,11 +1,12 @@
+const { EPS } = require('../../constants')
+
 const FuzzyFactory = require('./FuzzyFactory')
-const { EPS } = require('../constants')
 
-const FuzzyShape2Factory = function () {
+const FuzzyFactory2d = function () {
   this.vertexfactory = new FuzzyFactory(2, EPS)
 }
 
-FuzzyShape2Factory.prototype = {
+FuzzyFactory2d.prototype = {
   getVertex: function (sourcevertex) {
     let elements = [sourcevertex[0], sourcevertex[1]]
     // FIXME: oh blimey, this actually never did anything : returns the SOURCE VERTEX!!
```

Comment:
I think this comment is off -- it should only return sourcevertex if it can't find elements.

If sourcevertex is of the correct form then it can probably be use directly.

It might be clearer to write something like this.

lookupOrCreate(elements, otherwise => sourcevertex)

Reply:
@kaosat-dev Should I move this code to moveOutOfCode? I don't think we will use this.

Reply:
I don't believe we will use FuzzyFactory2 so let's trash it early. :)

Reply:
Don't you need this to ensure unique vertices?